osu.Game.Rulesets.Osu.Tests/TestSceneSkinFallbacks.cs

```diff
@@ -80,15 +80,18 @@ public void TestBeatmapSkinDefault()
 
         public class CustomSkinWorkingBeatmap : ClockBackedTestWorkingBeatmap
         {
-            private readonly ISkinSource skin;
+            private readonly ISkinSource skinSource;
 
             public CustomSkinWorkingBeatmap(IBeatmap beatmap, Storyboard storyboard, IFrameBasedClock frameBasedClock, AudioManager audio, ISkinSource skin)
                 : base(beatmap, storyboard, frameBasedClock, audio)
             {
-                this.skin = skin;
+                if (!(skinSource is IBeatmapSkin))
+                    throw new ArgumentException("The provided skin source must be of type IBeatmapSkin.");
```

Comment:
this is completely bogus. you're checking the wrong thing (`skinSource` which isn't assigned to yet, instead of `skin`), and even after correcting this test doesn't pass. did you try to run tests after making these changes at all?

Reply:
I'll take a second look about this.

Regarding testing, I forgot that that you can run the non-visual tests under Rider's Unit Tests tab, and I'm very sorry about that for forgetting this. I was told and used to run InspectCode.ps1 back then when running Windows.

Reply:
for clarification you can run *any* test, including the visual ones (headless), through the nunit runner.